com/uber/kafkaSpraynozzle/stats/StatsDReporter.java

```diff
@@ -0,0 +1,61 @@
+package com.uber.kafkaSpraynozzle.stats;
+
+import com.timgroup.statsd.NonBlockingStatsDClient;
+import com.timgroup.statsd.StatsDClient;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Statsd StatsReporter implementation
+ *
+ * To initialize this class you need to provide it the path to a .properties file containing:
```

Comment:
I'm a little conflicted on this. You needed multiple arguments to configure statsd, but because you don't have a guarantee that its statsd that's being initialized, you couldn't put them all in the args file, which is also getting pretty crowded with options.